data/index.html

```diff
@@ -192,13 +253,59 @@ <h2 class="demoHeaders">Dave's Fire NightDriverStrip</h2>
             {
                 effectIndex: i,
             });
+            RequestEffectList();
         }
 
         function NextStep()
         {
             // Add here when needed
         }
 
+        // UpdateEnabled
+        //
+        // Update the status on the page to reflect the currently enabled effects
+
+        function UpdateEnabled(effectData)
+        {
+            if (null == effectData)
+                return;
+            
+            $('#enabledCount').empty();
+            $('#enabledCount').append(text = "Effects Enabled: " + effectData.enabledCount + "/" + effectData.Effects.length);
+
+            if (effectData.enabledCount > 0)
+            {
+                $('#currentEffectProgress').css('background-color', 'green');
+            }
+            else
+            {
+                $('#currentEffectProgress').css('background-color', 'red');
+            }
+
+            $.each($('#effectList li input'), function(i, v)

```

Comment:
This section was intended to sync the checked state of all of the check boxes, so that if updates are made on another computer they will still be accurately represented on the web page after the next update. 

This seems to be causing odd behavior from my mobile browser, where checking the checkbox appears to be delayed until the next update happens. Sometimes the checked state will jump back and forth a couple of times before settling. This is also iterating over the list of all effects on the page, which could be time intensive if there is a large number of effects.

I'm not sure if there is a better alternative to accomplish this, or if it should be removed.

Reply:
I'm not sure I can really help you with making a decision about this. I can't test it myself; I don't have the hardware, nor the mobile browser in question. That said, on the surface it sounds like something that might be caused by some caching-related oddity, which may be specific to the browser that exhibits the behavior you describe.